README.md

```diff
@@ -13,9 +13,14 @@ and if you're planning to deploy then you'll need the [deployment repository](ht
 
 ## Development Server
 ### Boot2Docker
-> If you're developing on a Mac then Docker won't run natively, you'll be running 
-> a single VM with linux installed where your Docker containers run. To start the vm 
-> run the following first before continuing:
+> If you're developing on a Mac then Docker won't run natively, you'll be running a single VirtialMachine with linux installed where your Docker containers run. The first time you will need to create the VirtualMachine.
```

Comment:
There's a typo in the first instance of `VirtualMachine` (`VirtialMachine`), and “virtual machine” doesn't need capitalising because it's not a proper noun or anything. Also, can you wrap this as it was before?